esmvaltool/recipes/examples/recipe_check_obs.yml

```diff
@@ -572,7 +576,7 @@ diagnostics:
         mip: Oyr
     additional_datasets:
       - {dataset: WOA, project: OBS, tier: 2,
-         type: clim, version: 2013v2, start_year: 2000, end_year: 2000}
+         type: clim, version: 2018, start_year: 2315, end_year: 2315}
     scripts: null
```

Comment:
The last four variables are not part of the new cmorizer script with the version 2018 of the data. Keeping these variables here would break the recipe. Would it be possible to add the version 2013 of the datasets for the variables that are not part of the cmorizer? Otherwise, files like `OBS_WOA_clim_2018_Oyr_o2*` will not be found and files like `OBS_WOA_clim_2013v2_Oyr_o2*` will not be called.